charts/tenant-base/chart/values.yaml

```
@@ -0,0 +1,72 @@
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+
+replicaCount: 1
+
+livenessProbe:
+  httpGet:
+    path: /
+    port: http
+
+readinessProbe:
+  httpGet:
+    path: /
+    port: http
+
+volumes:
+  - name: cache-volume
+    emptyDir: {}
+
+volumeMounts:
+  - mountPath: /cache
+    name: cache-volume
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+## Enable an Specify container in extraContainers.
+extraContainers: ""
+# extraContainers: |
+#   - name: extra-container
+#     image: stefanprodan/podinfo:latest
+#     ports:
+#       - name: http
+#         containerPort: 80
+
+## Volumes that can be used in init containers that will not be mounted to deployment pods
+extraContainerVolumes: []
+  # - name: empty-dir-volume
+  #   emptyDir: {}
+
+service:
+  type: ClusterIP
+
+ports:
+  - podPortName: http
+    podPort: 80
+    servicePortName: http
+    servicePort: 80
+    protocol: TCP
+  - podPortName: https
+    podPort: 443
+    servicePortName: https
+    servicePort: 443
+    protocol: TCP
```

Make it work with podinfo:
```suggestion
  - podPortName: http
    podPort: 9898
    servicePortName: http
    servicePort: 80
    protocol: TCP
```